dask-gateway-server/dask_gateway_server/auth.py

```diff
@@ -376,13 +375,10 @@ async def authenticate(self, request):
         if token is None:
             raise unauthorized("jupyterhub")
 
-        url = "{}/authorizations/token/{}".format(
-            self.jupyterhub_api_url,
-            quote(token, safe=""),
-        )
+        url = f"{self.jupyterhub_api_url}/user"
 
         kwargs = {
-            "headers": {"Authorization": "token %s" % self.jupyterhub_api_token},
```

Comment:
Notably, `jupyterhub_api_token` is entirely not needed by JupyterHubAuth. With this implementation, dask-gateway technically doesn't need to be registered as a jupyterhub service, since it makes no use of the jupyterhub API _as itself_. That would only really be needed if the gateway began applying full oauth to protect dashboard pages, which appear to currently only be protected by the unguessable cluster names.